### ISSUE TITLE
Landscape view is unusable

### ISSUE THREAD

**Issue by moshner** [ripjar/material-datetime-picker#85]
This is a very pretty date picker but when I open it on a phone in landscape view it becomes unusable. The buttons fall off the bottom and can't be seen.
![graphic_8_9_2016 11_01_12 am](https://cloud.githubusercontent.com/assets/3300206/17523538/9fea00dc-5e20-11e6-8ffb-a4c0d6460eb7.jpg)

**Comment by Jivings** [comment]
Yep, I should create a landscape version. It needs some thinking about but probably something like the one on the guidelines page [here][1].

[1]: https://material-design.storage.googleapis.com/publish/material_v_8/material_ext_publish/0B3321sZLoP_Hamc5d1k2SmhlV2c/components_pickers_date3.png

**Comment by josdejong** [comment]
Would love to see a mobile friendly version of `material-datetime-picker` too.

Maybe a first simple step would be to make the date picker vertically scrollable when it doesn't fit, that would at least make it usable on mobile screens.

**Comment by El-Mahbub** [comment]
I have a same issue but i added scrollable and media screen, maybe this can solve this issue :

```
@media (max-height: 512px){
  .c-datepicker{
    max-height: 400px;
    max-width: 300px;
    overflow-y: auto;
    overflow-x: hidden;
  }
}
@media (max-height: 320px){
  .c-datepicker{
    max-height: 320px;
    max-width: 240px;
    overflow-y: auto;
    overflow-x: hidden;
  }
}
```
